src/openfermion/transforms/_code_transform_functions_test.py

```diff
@@ -36,6 +38,26 @@ def lih_hamiltonian():
 
 
 class CodeTransformTest(unittest.TestCase):
+    def test_tranform_function(self):
+        ferm_op = FermionOperator('2')
+        n_modes = 5
+        qubit_op = binary_code_transform(ferm_op, parity_code(n_modes))
+        self.assertEqual(qubit_op.terms,
+                         {((1, 'Z'), (2, 'X'),(3, 'X'),(4, 'X')): 0.5,
+                          ((2, 'Y'),(3, 'X'), (4, 'X')): 0.5j})
+        ferm_op = FermionOperator('2^')
+        n_modes = 5
+        qubit_op = binary_code_transform(ferm_op, parity_code(n_modes))
+        self.assertEqual(qubit_op.terms,
```

Comment:
I meant these two tests too (instead of asserting something about `qubit_op.terms`, create a new QubitOperator called `correct_op` and do `self.assertTrue(qubit_op.isclose(correct_op))`.

Reply:
ok I got a bunch of those in other tests I think, will change it all then.